db/schema.rb

```diff
@@ -26,19 +26,30 @@
 
   create_table "items", force: :cascade do |t|
     t.decimal "price"
-    t.integer "stock_level"
+    t.integer "status"
     t.string "image"
     t.string "name"
     t.text "description"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  create_table "order_items", force: :cascade do |t|
+    t.bigint "item_id"
+    t.bigint "order_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "quantity"
```

Comment:
I don't really think this should be here. We can get this info through the relationship, without the extra column on the join.

Reply:
Are you saying that if we fill the cart with three same items, and checkout that each would be it's own row on the OrderItem table?

Reply:
Yeah I think that we could move that to work at the model level--code smell here seems to be off to me.